packages/flutter_tools/test/commands/create_test.dart

```diff
@@ -57,1064 +57,1067 @@ void main() {
     tryToDelete(tempDir);
   });
```

Comment:
Here I would place the tag on the file instead of adding another layer of nesting